swagger.yaml

```
@@ -2153,6 +2157,13 @@ components:
         - NOT_PAID
         - ENDED
 
+    SubscriptionType:
+      type: string
+      description: Type of subscription
+      enum:
+        - STRIPE
+        - APPLE
```

```suggestion
        - APPLE_IAP
```
In future we could also add the ```Android IAP support```